src/styles/table.styl

```diff
@@ -104,49 +104,53 @@
     padding em(15px) em(8px)
 
 .fil-file-audio
-  background embedurl("../assets/icons/icon-type-audio.svg") 0 center no-repeat
+    background embedurl("../assets/icons/icon-type-audio.svg") 0 center no-repeat
```

Comment:
My turn:  double quotes! Lots of double quotes!

(it makes me stupidly happy to catch that 😛 )